stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel_actions/ChannelMembersAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package io.getstream.chat.android.ui.channel_actions
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import io.getstream.chat.android.client.models.Member
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.ui.databinding.StreamItemChannelMemberBinding
+
+internal class ChannelMembersAdapter(
+    private val onMemberClicked: (Member) -> Unit
+) : ListAdapter<Member, ChannelMemberViewHolder>(
+    object : DiffUtil.ItemCallback<Member>() {
+        override fun areItemsTheSame(oldItem: Member, newItem: Member): Boolean {
+            return oldItem.user.id == newItem.user.id
+        }
+
+        override fun areContentsTheSame(oldItem: Member, newItem: Member): Boolean {
+            return oldItem == newItem
+        }
+    }
```

Comment:
It would simplify this class a bit if we named this `object`, and nested it inside the adapter, instead of declaring it inline

Reply:
Done